spec/book_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../book'
+require 'rspec'
+
+describe Book do
+  let(:book) { Book.new('Ring Shout', 'FP. P. Djèlí Clark') }
+
+  describe 'creating a new book' do
+    it 'should have the correct properties' do
+      expect(book.title).to eq('Ring Shout')
+      expect(book.author).to eq('FP. P. Djèlí Clark')
+      expect(book.rentals).to be_an(Array)
+      expect(book.rentals).to be_empty
+    end
+  end
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍